tsconfig.json

```diff
@@ -17,11 +17,12 @@
     "resolveJsonModule": true,
     "isolatedModules": true,
     "noEmit": true,
-    "jsx": "react",
+    "jsx": "react-jsx",
```

Comment:
for using the latest JSX features, we have to start using `react-jsx` or typescript won't be able to identify the jsx v2 syntax which is supported in react already.